### PR TITLE
Feat/billing debt nic

### PR DESCRIPTION
**Pull request by marie-j**
### Description of the Change

Display message to inform that only NIC Billing can pay debt 

ref: MANAGER-1970

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-manager#629.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✔ 
  * Build manager EU ✘  
  * Build manager US ✔ 
  * Tests ✔

File: client/app/account/billing/translations/Messages_fr_FR.xml [hunk 1]
```diff
@@ -74,6 +74,7 @@
    <translation id="autorenew_service_debt_before_paying_title" qtlid="506780">Information importante</translation>
    <translation id="autorenew_service_debt_before_paying_1" qtlid="498209">Avant de vous engager sur une nouvelle période de temps, nous vous invitons à d'abord régler votre dette restante.</translation>
    <translation id="autorenew_service_debt_before_paying_2" qtlid="498222">Rendez-vous dans l’espace facturation afin de régulariser la facturation pour le service <strong>{{service}}</strong>.</translation>
+   <translation id="autorenew_service_debt_nic_billing">Pour payer ce service, connectez-vous depuis votre compte client de facturation {{ nicBilling }}</translation>
```
Comment (FredericEspiau) [inline client/app/account/billing/translations/Messages_fr_FR.xml]:
```suggestion
   <translation id="autorenew_service_debt_nic_billing">Pour payer ce service, connectez-vous depuis votre compte client de facturation {{ nicBilling }}.</translation>
```

File: client/app/account/billing/autoRenew/warnNicBilling/billing-autoRenew-warnNicBilling.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div data-ng-controller="billingControllersAutoRenewWarnNicBillingCtrl as $ctrl">
```
Comment (FredericEspiau) [inline client/app/account/billing/autoRenew/warnNicBilling/billing-autoRenew-warnNicBilling.html]: Component ? :smile:

Comment (marie-j) [reply client/app/account/billing/autoRenew/warnNicBilling/billing-autoRenew-warnNicBilling.html]: I'm not sure I can switch to a component as it still uses the legacy modal system

**Pull request merged**